test/WebJobs.Script.Tests/Rpc/RpcMessageConversionExtensionsTests.cs

```diff
@@ -46,5 +48,31 @@ public void HttpObjects_Query(string queryString, string[] expectedKeys, string[
                 Assert.Equal(rpcRequestObject.Http.Query.GetValueOrDefault(expectedKeys[i]), expectedValues[i]);
             }
         }
+
+        [Theory]
+        [InlineData(BindingDirection.In, "blob", DataType.String)]
+        [InlineData(BindingDirection.In, "blob", DataType.Binary)]
+        [InlineData(BindingDirection.In, "blob", DataType.Stream)]
+        [InlineData(BindingDirection.Out, "blob", DataType.String)]
```

Comment:
one variataion for DataType.String, DataType.Binary and  DataType.Stream is enough